activities/data_selfie/views.py

```diff
@@ -18,12 +19,11 @@ class UploadView(PrivateMixin, DropzoneS3UploadFormView):
     success_url = reverse_lazy('my-member-data-selfie')
 
     def get_upload_to(self):
-        return ('member/{}/uploaded-data/data-selfie/{}/'
-                .format(self.request.user.id, int(time())))
+        return get_upload_dir(DataSelfieDataFile, self.request.user)
 
     def get_upload_to_validator(self):
-        return (r'^member/{}/uploaded-data/data-selfie/\d+/'
-                .format(self.request.user.id))
+        return (r'^member/{}/uploaded-data/{}/\d+/'.format(
+            self.request.user.id, DataSelfieDataFile._meta.app_label))
```

Comment:
let's make this into a `get_upload_dir_validator` with the same arguments as `get_upload_dir`?